types/fluxible-addons-react/connectToStores.d.ts

```diff
@@ -0,0 +1,31 @@
+import { ComponentContext } from 'fluxible'
+import { StoreClass } from 'dispatchr'
+import React = require('react')
+
+// Diff / Omit taken from https://github.com/Microsoft/TypeScript/issues/12215#issuecomment-311923766
+type Diff<T extends string, U extends string> = ({ [P in T]: P } & { [P in U]: never } & { [x: string]: never })[T];
+type Omit<T, K extends keyof T> = Pick<T, Diff<keyof T, K>>;
```

Comment:
Should this be changed to `Pick<T, Exclude<keyof T, K>>`, or is it too soon for 2.8